<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposito</class>
 <widget class="QMainWindow" name="deposito">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>685</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Deposito</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagen/logo.png</normaloff>imagen/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="2" column="2" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>653</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>338</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>208</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="generar_lista">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Generar lista al público</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagen/archivo_pdf.png</normaloff>imagen/archivo_pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="buscador_producto">
        <property name="placeholderText">
         <string>Buscar producto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buscar_producto">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Buscar producto</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/lupa.png</normaloff>imagen/lupa.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="eliminar_producto_2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Productos fallados</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagen/articulo_fallado.png</normaloff>imagen/articulo_fallado.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="backup_productos">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Backup en excel</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagen/backup.png</normaloff>imagen/backup.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="nuevo_producto">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Crear producto</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/signo_mas.png</normaloff>imagen/signo_mas.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modificar_producto">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Modificar producto</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagen/editar.png</normaloff>imagen/editar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTableWidget" name="tabla_deposito">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Cód.</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Proveedor</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rubro</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sub-Rubro</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marca</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descripción</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cód. Prov.</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Costo</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P. de venta</string>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buscador_producto</tabstop>
  <tabstop>buscar_producto</tabstop>
  <tabstop>nuevo_producto</tabstop>
  <tabstop>modificar_producto</tabstop>
  <tabstop>eliminar_producto_2</tabstop>
  <tabstop>tabla_deposito</tabstop>
  <tabstop>backup_productos</tabstop>
  <tabstop>generar_lista</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
